028-HubAndSpoke/README.md

```diff
@@ -0,0 +1,45 @@
+# What The Hack - Azure Networking with Hub and Spoke
+
+## Introduction
+
+In this WTH exercise students will engage into an increasingly complex Azure Virtual Network setup, finding many of the challenges that often appear in production environments. The Hack starts with a deep dive in hub and spoke networking and user-defined routes during the first day, and goes on by adding components such as the Azure Application Gateway or PaaS networking in the second day.
+
+The estimated duration time for this hack is 2 days.
+
+## Learning Objectives
+
+After completing this WTH, participants will be familiar with these concepts, amongst others:
+
+- VNet peering
+- User Defined Routes and hybrid connectivity
+- Scenarios with potential asymmetric routing
+- Azure Firewall DNAT (Destination Network Address Translation) rules
+- Azure Firewall network and application rules
+- Designs combining the Azure Firewall and the Application Gateway
+- PaaS Networking options such as Vnet service endpoints and Private Link
+
+## Prerequisites
+
+No specific prerequisites are needed, but familiarity with Azure Networking is a plus.
+
+## Contributors
+
+- Thomas Vuylsteke
+- Jose Moreno
+
+## Getting started
+
+Start this WhatTheHack with the [student guide](Student/README.md)

```

Comment:
This line should go away once you merge the two readme's.

Reply:
Done